src/Interfaces/Severity.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Sentry\Interfaces;
+
+final class Severity
```

Comment:
This will replace `Client::translateSeverity` and some more in the future, right?

Reply:
Yes, that's the idea.